### PR TITLE
[Feature/models] Models creation

### PR DESCRIPTION
**Pull request by vgeruso**
Criação dos models de acordo com a modelagem dos dados

### CODE REVIEW

**Comment by linzera** [comment]
Please, rename the models to English!

File: src/models/Personagem.js [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+    },
+    /**
+     * Raça do personagem
+     * Tem como base principal 4 raças:
+     * Humano, Anão, Elfo e Halfling
+     * Obs.: terão as raças acrescentadas, essas informações serão
+     * setadas no fromulario de criação do personagem no Frontend.
+     */
+    raca: {
+        type: String,
```
Comment (linzera) [inline src/models/Personagem.js]: I think race should be a model to.

File: src/models/Personagem.js [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+    },
+    /**
+     * Classe do personagem
+     * Tem como base principal 4 classes:
+     * Mago, Clerigo, Homem de Armas, Ladrão
+     * Obs.: Cada classe tem entre 3 e 4 subclasses
+     * cada uma de acordo com a o alinhamento escolhido.
+     */
+    classe: {
+        type: String,
```
Comment (linzera) [inline src/models/Personagem.js]: Class should be a model as well

Comment (VictorGeruso) [reply src/models/Personagem.js]: Okay, I'm going to change the data model, thinking about it, because it makes sense

**Pull request merged**